mod/host.py

```diff
@@ -1152,7 +1152,10 @@ def report_current_state(self, websocket):
             for symbol, value in pluginData['outputs'].items():
                 if value is None:
                     continue
-                websocket.write_message("output_set %s %s %f" % (pluginData['instance'], symbol, value))
+                if isinstance (value, float):
+                    websocket.write_message("output_set %s %s %f" % (pluginData['instance'], symbol, value))
+                if isinstance (value, str):
```

Comment:
The message needs to be saved so reloading the web page can re-send all the values.
The alternative would be to ask the plugin what its current state of a specific port + key is.